### PR TITLE
'Add reminder' doesn't work fix

### LINKED ISSUE #111: 'Add reminder' doesn't work
## How to reproduce

1. On 'user-todo-list' page press 'Add reminder' button for any existing todo.
2. Select any date and time in datepicker.
3. Press enter.

screen shot 2017-02-12 at 21 29 28

### PR DESCRIPTION
**Pull request by akireikin**
#111 fixed

### CODE REVIEW

**Comment by prolic** [comment]
Seems like you messed up some pushed commits.

**Comment by prolic** [comment]
Can you rebased against develop?

**Comment by akireikin** [comment]
I did it. It seems you have already fixed this issue in develop.

**Comment by akireikin** [comment]
The problem was in https://github.com/prooph/proophessor-do/commit/ea4ffdefb65942859745eb2d4b763cfbb7d01ba4#diff-5044c9fa2c209be2ffef808d5a49edbdR39

Anyway, at least I added a test :)

File: tests/Model/Todo/Command/AddReminderToTodoTest.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     */
+    public function it_creates_reminder_from_payload()
+    {
+        $command = new AddReminderToTodo(['reminder' => '2017-02-14T05:00:00.000Z']);
+        $reminder = $command->reminder();
+        $this->assertInstanceOf(TodoReminder::class, $reminder);
+        $this->assertTrue($reminder->isOpen());
+        $this->assertSame('2017-02-14T05:00:00+00:00', $reminder->toString());
+    }
+}
```
Comment (prolic) [inline tests/Model/Todo/Command/AddReminderToTodoTest.php]: New line at end of file missing.

File: tests/Model/Todo/Command/AddReminderToTodoTest.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This file is part of prooph/proophessor-do.
+ * (c) 2014-2016 prooph software GmbH <contact@prooph.de>
```
Comment (prolic) [inline tests/Model/Todo/Command/AddReminderToTodoTest.php]: It's 2017

**Comment by prolic** [comment]
Thanks :+1:

**Pull request merged**